astropy/io/fits/column.py

```diff
@@ -981,6 +1045,53 @@ def _verify_keywords(cls, name=None, format=None, unit=None, null=None,
             else:
                 invalid['dim'] = (dim, msg)
 
+        if coord_type is not None and coord_type != '':
+            msg = None
```

Comment:
I didn't really like this way of doing it, but then realized you are just repeating the style of the file, so all OK.